src/screens/Assistant/ResultItem.tsx

```diff
@@ -30,8 +29,13 @@ import insets from '@atb/utils/insets';
 import {getTranslatedModeName} from '@atb/utils/transportation-names';
 import {LegMode} from '@entur/sdk';
 import React from 'react';
-import {AccessibilityProps, View, ViewStyle} from 'react-native';
-import {ScrollView, TouchableOpacity} from 'react-native-gesture-handler';
+import {
+  AccessibilityProps,
+  View,
+  ViewStyle,
+  TouchableOpacity,
+} from 'react-native';
+import {ScrollView} from 'react-native-gesture-handler';
```

Comment:
Is it important where `ScrollView` is imported from? A quick search in the code now showed 19 imports from `react-native-gesture-handler` and 3 imports fra `react-native`.

Reply:
react-native-gesture-handler typically has better performance AFAIK, but in the case of Touchable*s it lacks accessibility. But for ScrollView it should be fine.